sample/android/src/main/java/co/nimblehq/kmm/sample/lib/TypeAlias.kt

```diff
@@ -0,0 +1,3 @@
+package co.nimblehq.kmm.sample.lib
+
+typealias IsLoading = Boolean
```

Comment:
This naming is weird, where is the use case of it? 🤔

Reply:
@markgravity It's an alias of the loading state. We're using it in Android Templates and applied to KMM Templates too https://github.com/nimblehq/kmm-templates/blob/9bf253c53a000a99f6ef59b15251a7fb532d4af7/sample/android/src/main/java/co/nimblehq/kmm/template/ui/base/BaseViewModel.kt#L17-L18